### PR TITLE
Remove calls to AppStart from normal Activities

### LINKED ISSUE #2810: Failed to resolve type MvvmCross.ViewModels.IMvxAppStart
### Steps to reproduce :scroll:
```
Xamarin caused by: android.runtime.JavaProxyThrowable: MvvmCross.Exceptions.MvxIoCResolveException: Failed to resolve type MvvmCross.ViewModels.IMvxAppStart
MvvmCross.IoC.MvxIoCContainer.Resolve(Type t)<a1ab21c493b64a15919bccd6a6fb7be2>:0
MvvmCross.IoC.MvxIoCContainer.Resolve<T>()<a1ab21c493b64a15919bccd6a6fb7be2>:0
MvvmCross.IoC.MvxIoCProvider.Resolve<T>()<a1ab21c493b64a15919bccd6a6fb7be2>:0
MvvmCross.Mvx.Resolve<TService>()<a1ab21c493b64a15919bccd6a6fb7be2>:0
MvvmCross.Platforms.Android.Views.MvxActivityViewExtensions.OnViewDestroy(IMvxAndroidView androidView)<a1ab21c493b64a15919bccd6a6fb7be2>:0
MvvmCross.Platforms.Android.Views.MvxActivityAdapter.EventSourceOnDestroyCalled(object sender, EventArgs eventArgs)<a1ab21c493b64a15919bccd6a6fb7be2>:0
at (wrapper delegate-invoke) <Module>.invoke_void_object_EventArgs(object,System.EventArgs)
MvvmCross.Base.MvxDelegateExtensions.Raise(EventHandler eventHandler, object sender)<a1ab21c493b64a15919bccd6a6fb7be2>:0
MvvmCross.Droid.Support.V7.AppCompat.EventSource.MvxEventSourceAppCompatActivity.OnDestroy()<a463012b56d64d1f9cc54a527e8dd780>:0
MvvmCross.Droid.Support.V7.AppCompat.MvxAppCompatActivity.OnDestroy()<a463012b56d64d1f9cc54a527e8dd780>:0
Android.App.Activity.n_OnDestroy(IntPtr jnienv, IntPtr native__this)<8e97598d39014e64a9b297bda6092daf>:0
at (wrapper dynamic-method) System.Object.51564007-77ee-4aca-b000-e0c439692f3c(intptr,intptr)
mvvmcross.droid.support.v7.appcompat.MvxAppCompatActivity.n_onDestroy(Native Method)
mvvmcross.droid.support.v7.appcompat.MvxAppCompatActivity.onDestroy()MvxAppCompatActivity.java:61
android.app.Activity.performDestroy()Activity.java:7462
android.app.Instrumentation.callActivityOnDestroy()Instrumentation.java:1255
android.app.ActivityThread.performDestroyActivity()ActivityThread.java:4590
android.app.ActivityThread.handleDestroyActivity()ActivityThread.java:4621
android.app.ActivityThread.-wrap5()Unknown Source:0
android.app.ActivityThread$H.handleMessage()ActivityThread.java:1757
android.os.Handler.dispatchMessage()Handler.java:105
android.os.Looper.loop()Looper.java:164
android.app.ActivityThread.main()ActivityThread.java:6938
java.lang.reflect.Method.invoke(Native Method)
com.android.internal.os.Zygote$MethodAndArgsCaller.run()Zygote.java:327
com.android.internal.os.ZygoteInit.main()ZygoteInit.java:1374

 public SplashScreen() : base(Resource.Layout.SplashScreen)
 {
        this.RegisterSetupType<Setup>();
 }

 protected override void RunAppStart(Bundle bundle)
 {
        var starter = MvvmCross.Mvx.Resolve<IMvxAppStart>();
        starter.Start(model);
  }
```

### Configuration :wrench:

**Version:** 
MvvmCross 6.0.0

**Platform:** 
Android 5.1.1, 8.0.0
SAMSUNG SM-G5500
SAMSUNG SM-G950F

### LINKED ISSUE #3014: App not starting properly when App is start via Activity
## 🐛 Bug Report
I have not been able to reproduce this bug in my development environment, but our production app are generating crash reports that Mvx.Resolve - i.e stuff that should resolve is returning null.

I know in Mvx 6.x the startup logic was changed and in particular I'm referring to this code here:
https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross.Android.Support/V7.AppCompat/MvxAppCompatActivity.cs

 ```
protected override void OnCreate(Bundle bundle)
        {
            base.OnCreate(bundle);
            ViewModel?.ViewCreated();

            if(!(this is IMvxSetupMonitor))
            {
                RunAppStart(bundle);
            }
        }

```

The above code to my understanding (from reading documentation etc.) is kind of a catch all to ensure that the app is started properly if not entering the app via Android splash screen, like for example if entering the app via a deep link.

So I have two concerns:

1. Since AppStart only runs **after** ViewCreated, a user of Mvx cannot safely do Mvx.Resolve when overriding VM.ViewCreated, so shouldn't ViewCreated be called after AppStart?

1. And here perhaps I'm not understanding this properly, but the following line of code implies the VM is instantiated, meaning the the VM's constructor has been invoked with ioc parameters, but if AppStart runs after ViewCreated then those ioc parameters won't be available yet. 
 
          `  ViewModel?.ViewCreated();`

### Expected behavior
That when I do an Mvx.Resolve in a vm in Android it should resolve if part of AppStart.

When starting via MvxSplashScreenAppCompatActivity this doesn't seem to be a problem.
But when start via Activity MvxAppCompatActivity, I think there could be problems

### Reproduction steps

Haven't figured this out yet.

### Configuration

**Version:** 6.0.1

**Platform:** 
- [x ] :robot: Android

### LINKED ISSUE #3177: CustomAppStart methods called twice
## 🐛 Bug Report

_CustomAppStart_ methods is called twice, because splash screen _OnDestroy_ sets _MvxAppStart.IsStarted_ property to **false**, when application _CurrentTopActivity_ is **null**.

### Expected behavior
I expect _CustomAppStart_ methods to be executed only once.

### Reproduction steps
1. When the application starts, a splash screen is displayed and the methods of the _MvxAppStart_ class are executed, in particular, _NavigateToFirstViewModel_, also the _IsStarted_ flag is set to **true**.
2. _NavigateToFirstViewModel_, in turn, shows the login page (_MvxAppCompatActivity_).
3. When any _MvxAppCompatActivity_ is created, a check for _IsStarted_ is performed. If it is **false**, the application start methods mentioned above are executed.
4. When login page Activity is displayed, the _IsStarted_ flag is already set to **true**, so nothing happens.
5. E.g. system dialog for application permissions shows up in front of login page.
6. The same time, the _OnDestroy_ method of the splash screen is called. _MvxActivityViewExtensions.OnViewDestroy_ checks for _MvxAndroidCurrentTopActivity_ which is **null**, so the _IsStarted_ flag is set to **false**.
7. After login, another page is displayed, so the newly created _MvxAppCompatActivity_ checks for _IsStarted_ and calls _MvxAppStart_ methods again.

The same problem can be reproduced by pushing an application to background at right moment, so _MvxAndroidCurrentTopActivity_ is **null**.

### Configuration

**Version:** 6.2.1

**Platform:** 
- [ ] :iphone: iOS
- [x] :robot: Android
- [ ] :checkered_flag: WPF
- [ ] :earth_americas: UWP
- [ ] :apple: MacOS
- [ ] :tv: tvOS
- [ ] :monkey: Xamarin.Forms

### LINKED ISSUE #3196: First ViewModel being created twice on Android
## 🐛 Bug Report

When using the default `RegisterAppStart<ViewModel>` and the default `MvxActivity<ViewModel>` for the main launcher, that first viewmodel is created twice.

Maybe this is a regression? A similar bugreport was submitted for a previous version here #2038.

### Expected behavior

The first ViewModel should only be created once.

### Reproduction steps

I have created a sample project to reproduce the bug [here](https://github.com/slarti-zak/MvxDuplicateAppStartViewModel).

### Configuration

**Version:** 6.2.2

**Platform:** 
- [ ] :iphone: iOS
- [x] :robot: Android
- [ ] :checkered_flag: WPF
- [ ] :earth_americas: UWP
- [ ] :apple: MacOS
- [ ] :tv: tvOS
- [ ] :monkey: Xamarin.Forms

### PR DESCRIPTION
**Pull request by nmilcoff**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)
Bug fix

### :arrow_heading_down: What is the current behavior?

As described in #3196, when an Android app does not use a SplashScreen, the first ViewModel is created twice:
- One time through the MvxAppStart mechanism
- One time in the Activity's `base.OnCreate` method

Also `Mvx.IoCProvider.Resolve` is used in SplashScreen classes and it's causing some crashes as described in #2810.

### :new: What is the new behavior (if this is a feature change)?

AppStart code is removed from normal activities. As a result, when an Android app does not use a SplashScreen the AppStart mechanism won't run, but this is the best experience we can achieve right now.

### :boom: Does this PR introduce a breaking change?

Probably, but it's for the greater good if fixing the issue (also I don't think this will have a major impact).

### :bug: Recommendations for testing

Comment out the SplashScreen in Playground.Droid and set `RootView` as `MainLauncher`. Set a breakpoint in `RootViewModel` - it should be called only once.

### :memo: Links to relevant issues/docs
Fixes #3196.
Fixes #3177 (no more double initial navigation)
Closes #3014.
Fixes #2810.

### :thinking: Checklist before submitting

- [X] All projects build
- [X] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [ ] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contributing/mvvmcross-docs-style-guide))
- [X] Rebased onto current develop

### CODE REVIEW

**Comment by nickrandolph** [comment]
I'm not sure this is a great solution. As I understand it the issue is that on Android the startup activity (when no splash screen) is the same as the first view. On other platforms this isnt the case so we don't see this issue.
How was this addressed in the past? Is this issue present if the splash screen is used?

**Comment by nmilcoff** [comment]
@nickrandolph thanks for your review! In the past, this worked they way it will work if this PR is merged.

I understand and share your concern. As I mentioned in the description, I think this is the best experience we can provide given the current situation.

> Is this issue present if the splash screen is used?

In theory, no. But there are some issues like the explained in #3177 which indicates the current state is buggy.

**Comment by PicOLinO** [comment]
@nickrandolph, @nmilcoff buggy state with using splash screen confirmed. 
Sometimes I have a "floating" error with twice viewmodel initialize. Usually reproduced while app in background (in memory) and then trying to foreground (prehaps Android OS killed app in memory) . Initialization of first view model calling twice.
Firstly - from OnCreate method of activity, backgrounded on.
Secondly - from NavigateToFirstViewModel.

Do you recommended to create new issue for this problem?

Call stack of first initialize:
```
at System.Environment.get_StackTrace () [0x00000] in <c872650e44c44f97963d0388e1f6175a>:0 
  at SharedPcl.ViewModelsNew.MyViewModel+<Initialize>d__98.MoveNext () [0x00000] in <bf3904ded92344febe1090fbae026dff>:0 
  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine] (TStateMachine& stateMachine) [0x00000] in <c872650e44c44f97963d0388e1f6175a>:0 
  at SharedPcl.ViewModelsNew.MyViewModel.Initialize () [0x00000] in <bf3904ded92344febe1090fbae026dff>:0 
  at MvvmCross.ViewModels.MvxDefaultViewModelLocator+<>c__DisplayClass14_0.<RunViewModelLifecycle>b__0 () [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.ViewModels.MvxNotifyTask.Create (System.Func`1[TResult] asyncAction, System.Action`1[T] onException) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.ViewModels.MvxDefaultViewModelLocator.RunViewModelLifecycle (MvvmCross.ViewModels.IMvxViewModel viewModel, MvvmCross.ViewModels.IMvxBundle parameterValues, MvvmCross.ViewModels.IMvxBundle savedState, MvvmCross.Navigation.EventArguments.IMvxNavigateEventArgs navigationArgs) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.ViewModels.MvxDefaultViewModelLocator.Load (System.Type viewModelType, MvvmCross.ViewModels.IMvxBundle parameterValues, MvvmCross.ViewModels.IMvxBundle savedState, MvvmCross.Navigation.EventArguments.IMvxNavigateEventArgs navigationArgs) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.ViewModels.MvxViewModelLoader.LoadViewModel (MvvmCross.ViewModels.MvxViewModelRequest request, MvvmCross.ViewModels.IMvxBundle savedState, MvvmCross.Navigation.EventArguments.IMvxNavigateEventArgs navigationArgs) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.Platforms.Android.Views.MvxAndroidViewsContainer.ViewModelFromRequest (MvvmCross.ViewModels.MvxViewModelRequest viewModelRequest, MvvmCross.ViewModels.IMvxBundle savedState) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.Platforms.Android.Views.MvxAndroidViewsContainer.CreateViewModelFromIntent (Android.Content.Intent intent, MvvmCross.ViewModels.IMvxBundle savedState) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.Platforms.Android.Views.MvxAndroidViewsContainer.Load (Android.Content.Intent intent, MvvmCross.ViewModels.IMvxBundle savedState, System.Type viewModelTypeHint) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.Platforms.Android.Views.MvxActivityViewExtensions.LoadViewModel (MvvmCross.Platforms.Android.Views.IMvxAndroidView androidView, MvvmCross.ViewModels.IMvxBundle savedState) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.Platforms.Android.Views.MvxActivityViewExtensions+<>c__DisplayClass1_0.<OnViewCreate>b__1 () [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.Views.MvxViewExtensions.OnViewCreate (MvvmCross.Views.IMvxView view, System.Func`1[TResult] viewModelLoader) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.Platforms.Android.Views.MvxActivityViewExtensions.OnViewCreate (MvvmCross.Platforms.Android.Views.IMvxAndroidView androidView, Android.OS.Bundle bundle) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.Platforms.Android.Views.MvxActivityAdapter.EventSourceOnCreateCalled (System.Object sender, MvvmCross.Base.MvxValueEventArgs`1[T] eventArgs) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.Base.MvxDelegateExtensions.Raise[T] (System.EventHandler`1[TEventArgs] eventHandler, System.Object sender, T value) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.Droid.Support.V7.AppCompat.EventSource.MvxEventSourceAppCompatActivity.OnCreate (Android.OS.Bundle bundle) [0x00000] in <6f4d53f489084371969126ea5e21accf>:0 
  at MvvmCross.Droid.Support.V7.AppCompat.MvxAppCompatActivity.OnCreate (Android.OS.Bundle bundle) [0x00000] in <6f4d53f489084371969126ea5e21accf>:0 
  at ProjectMateAndroid.Activity.Supers.ViewBase`1[T].OnCreate (Android.OS.Bundle bundle) [0x00000] in <cdbaa058a7c645cd9ae0d072bc2663ae>:0 
  at ProjectMateAndroid.Activity.Supers.ViewWithMenuBase`1[T].OnCreate (Android.OS.Bundle bundle) [0x00000] in <cdbaa058a7c645cd9ae0d072bc2663ae>:0 
  at ProjectMateAndroid.Activity.MyView.OnCreate (Android.OS.Bundle bundle) [0x00000] in <cdbaa058a7c645cd9ae0d072bc2663ae>:0 
  at Android.App.Activity.n_OnCreate_Landroid_os_Bundle_ (System.IntPtr jnienv, System.IntPtr native__this, System.IntPtr native_savedInstanceState) [0x00000] in <cabb16d78fa04566879d989a34f4a132>:0 
  at System.Object.12 (System.IntPtr , System.IntPtr , System.IntPtr ) [0x00000] in <c872650e44c44f97963d0388e1f6175a>:0
```
Call stack of second initialize:
```
at System.Environment.get_StackTrace () [0x00000] in <c872650e44c44f97963d0388e1f6175a>:0 
  at SharedPcl.ViewModelsNew.MyViewModel+<Initialize>d__98.MoveNext () [0x00000] in <bf3904ded92344febe1090fbae026dff>:0 
  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine] (TStateMachine& stateMachine) [0x00000] in <c872650e44c44f97963d0388e1f6175a>:0 
  at SharedPcl.ViewModelsNew.MyViewModel.Initialize () [0x00000] in <bf3904ded92344febe1090fbae026dff>:0 
  at MvvmCross.ViewModels.MvxDefaultViewModelLocator+<>c__DisplayClass14_0.<RunViewModelLifecycle>b__0 () [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.ViewModels.MvxNotifyTask.Create (System.Func`1[TResult] asyncAction, System.Action`1[T] onException) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.ViewModels.MvxDefaultViewModelLocator.RunViewModelLifecycle (MvvmCross.ViewModels.IMvxViewModel viewModel, MvvmCross.ViewModels.IMvxBundle parameterValues, MvvmCross.ViewModels.IMvxBundle savedState, MvvmCross.Navigation.EventArguments.IMvxNavigateEventArgs navigationArgs) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.ViewModels.MvxDefaultViewModelLocator.Load (System.Type viewModelType, MvvmCross.ViewModels.IMvxBundle parameterValues, MvvmCross.ViewModels.IMvxBundle savedState, MvvmCross.Navigation.EventArguments.IMvxNavigateEventArgs navigationArgs) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.ViewModels.MvxViewModelLoader.LoadViewModel (MvvmCross.ViewModels.MvxViewModelRequest request, MvvmCross.ViewModels.IMvxBundle savedState, MvvmCross.Navigation.EventArguments.IMvxNavigateEventArgs navigationArgs) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.Navigation.MvxNavigationService+<Navigate>d__53.MoveNext () [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1[TResult].Start[TStateMachine] (TStateMachine& stateMachine) [0x00000] in <c872650e44c44f97963d0388e1f6175a>:0 
  at MvvmCross.Navigation.MvxNavigationService.Navigate (System.Type viewModelType, MvvmCross.ViewModels.IMvxBundle presentationBundle, System.Threading.CancellationToken cancellationToken) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.Navigation.MvxNavigationService.Navigate[TViewModel] (MvvmCross.ViewModels.IMvxBundle presentationBundle, System.Threading.CancellationToken cancellationToken) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at SharedPcl.CustomAppStart.NavigateToFirstViewModel (System.Object hint) [0x00000] in <bf3904ded92344febe1090fbae026dff>:0 
  at MvvmCross.ViewModels.MvxAppStart+<StartAsync>d__5.MoveNext () [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine] (TStateMachine& stateMachine) [0x00000] in <c872650e44c44f97963d0388e1f6175a>:0 
  at MvvmCross.ViewModels.MvxAppStart.StartAsync (System.Object hint) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.ViewModels.MvxAppStart.Start (System.Object hint) [0x00000] in <103d59106f92414bb3d9c03f45250bec>:0 
  at MvvmCross.Droid.Support.V7.AppCompat.MvxAppCompatActivity.RunAppStart (Android.OS.Bundle bundle) [0x00000] in <6f4d53f489084371969126ea5e21accf>:0 
  at MvvmCross.Droid.Support.V7.AppCompat.MvxAppCompatActivity.OnCreate (Android.OS.Bundle bundle) [0x00000] in <6f4d53f489084371969126ea5e21accf>:0 
  at ProjectMateAndroid.Activity.Supers.ViewBase`1[T].OnCreate (Android.OS.Bundle bundle) [0x00000] in <cdbaa058a7c645cd9ae0d072bc2663ae>:0 
  at ProjectMateAndroid.Activity.Supers.ViewWithMenuBase`1[T].OnCreate (Android.OS.Bundle bundle) [0x00000] in <cdbaa058a7c645cd9ae0d072bc2663ae>:0 
  at ProjectMateAndroid.Activity.MyView.OnCreate (Android.OS.Bundle bundle) [0x00000] in <cdbaa058a7c645cd9ae0d072bc2663ae>:0 
  at Android.App.Activity.n_OnCreate_Landroid_os_Bundle_ (System.IntPtr jnienv, System.IntPtr native__this, System.IntPtr native_savedInstanceState) [0x00000] in <cabb16d78fa04566879d989a34f4a132>:0 
  at System.Object.12 (System.IntPtr , System.IntPtr , System.IntPtr ) [0x00000] in <c872650e44c44f97963d0388e1f6175a>:0 
```

**Comment by Nickolas-** [comment]
What about to just create a special activity for such cases ? Let's say StartActivity that would handle such circumstances ?

**Comment by nmilcoff** [comment]
I don't think that will work @Nickolas-, at least it won't be super clean to implement. The main issue is that the first ViewModel is created twice in the scenario where the app has no splashscreen (once in AppStart, once in `base.OnCreate`). 

I guess to follow your idea we will have to check within the framework if the Activity is of a certain type

**Comment by Nickolas-** [comment]
It always hard to play with the state. You will run into several issues.

1. Multiple scenario when we invoke first ViewModel (Login/Logout) 
2. Handle multiple creation ViewModel 

It's easier just adapt one Activity type to initial process. It's give the clean vision of process. If we add extra logic for handling states it would be overkill.

**Comment by nmilcoff** [comment]
@martijn00 as far as I can see, this PR introduces a better approach for handling the startup scenario (and actually fixes the behavior - last week I spoke to a group of users who are not upgrading their Mvx version because of the duplicated initial ViewModel).

Can you please describe a bit more your concerns? 

I do agree however this should be managed carefully.

File: MvvmCross/Platforms/Android/Views/MvxSplashScreenActivity.cs [hunk 1]
```diff
@@ -88,9 +88,16 @@ public virtual async Task InitializationComplete()
 
         protected virtual async Task RunAppStartAsync(Bundle bundle)
         {
-            var startup = Mvx.IoCProvider.Resolve<IMvxAppStart>();
-            if (!startup.IsStarted)
-                await startup.StartAsync(GetAppStartHint(bundle));
+            if (Mvx.IoCProvider.TryResolve(out IMvxAppStart startup))
+            {
+                if (!startup.IsStarted)
```
Comment (Cheesebaron) [inline MvvmCross/Platforms/Android/Views/MvxSplashScreenActivity.cs]: If `IsStarted` is true, shouldn't we call `Finish()` on the Splash Screen? There is no reason to keep it open?

Comment (Nickolas-) [reply MvvmCross/Platforms/Android/Views/MvxSplashScreenActivity.cs]: In my case I call `finish ()`.

Comment (nmilcoff) [reply MvvmCross/Platforms/Android/Views/MvxSplashScreenActivity.cs]: True! I'll make that change, also for the AppCompat version

**Comment by JPSiller** [comment]
Hi all, 

Do you know if these changes will address the original issue (removing the appstart calls from normal activities) and do you think 6.2.3 will be released before Feb 28? We migrated from MvvmCross 4 to 6.2.2 and we have a planned release first week of March so I'm looking to know if this will be released before that and I'll be able to get those fixes or if there is a workaround I could manually implement such as downgrading to another mvvmcross 6 version that doesn't have this issue. Any suggestions?

Thanks!

**Comment by Cheesebaron** [comment]
@JPSiller this change didn't make it into 6.2.3. However, as you may have noticed, the method affected in this PR are all marked `virtual`. So you could override the methods in your own code as a workaround for now and applying the code there.

**Comment by JPSiller** [comment]
I implemented the workaround form https://github.com/MvvmCross/MvvmCross/issues/3268 @tbalcom successfully for now. Just wanted to reference it here too. Thanks for everyones help.

**Comment by nmilcoff** [comment]
Are we ok to merge this? @MvvmCross/core

**Pull request merged**